Dockerfile

```diff
@@ -13,25 +13,25 @@ RUN npm install
 
 COPY runit /etc/service/node-app
 RUN chmod -R 755 /etc/service/node-app
-RUN npm install -g supervisor
+RUN npm install -g nodemon
 
 # Set correct environment variables.
 # ENV HOME /opt/application
 
```

Comment:
Add a comment explaining that the default values in Dockerfile are relevant to production. In dev we will always be using docker-compose, anyway so values there will override the ENV values in Dockerfile. By having Dockerfile values be sensible production values, you can easily run the container in prod, without using docker-compose (which is nice)